### PR TITLE
ci: Merging 3.10.1 following release.

### PR DESCRIPTION
**Pull request by marksweb**
This merges `release/3.10.x` following the release of 3.10.1

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -226,7 +253,7 @@ Thanks to all contributors for their efforts!
 * Fixed an issue in ``wizards/create.html`` where the error message did not use the plural form
 * Improved documentation building
 * Updated the content for django CMS’s development community
-* Replaced all occurrences of ``force_text`` and ``smart_text`` against
+* Replaced all occurences of ``force_text`` and ``smart_text`` against
```
Comment (mazulo) [inline CHANGELOG.rst:256]: nit: maybe a typo here?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1135,7 +1162,7 @@ Thanks to all contributors for their efforts!
 ==================
 
 - Add require_parent option to CMS_PLACEHOLDER_CONF
-- Fix django-mptt version dependency to be PEP440 compatible
+- Fix django-mptt version depenency to be PEP440 compatible
```
Comment (mazulo) [inline CHANGELOG.rst:1165]:
```suggestion
- Fix django-mptt version dependency to be PEP440 compatible
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1078,7 +1105,7 @@ Thanks to all contributors for their efforts!
 - Improves redirection as a result of changes to page slugs, etc.
 - Improves performance of "watched models"
 - Improves frontend performance relating to resizing the sideframe
-- Corrects an issue where items might not be visible in structure mode menus
+- Corrects an issue where items might not be visible in structue mode menus
```
Comment (mazulo) [inline CHANGELOG.rst:1108]: nit: maybe a typo here? Maybe should be structure
```suggestion
- Corrects an issue where items might not be visible in structure mode menus
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1163,7 +1190,7 @@ Thanks to all contributors for their efforts!
 - Fixes PageField to work in Django 1.7 environments
 - Updates to community and project governance documentation
 - Added list of retired core developers
-- Added branch policy documentation
+- Added branch policy documentaion
```
Comment (mazulo) [inline CHANGELOG.rst:1193]:
```suggestion
- Added branch policy documentation
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -1373,7 +1400,7 @@ Please see Install/2.4 release notes *before* attempting to upgrade to version 2
 - CMS_FRONTEND_LANGUAGES limits django languages as well during language selection
 - Wymeditor updated to 1.0.4a
 - icon_url escape fixed
-- Ukrainian translation added
+- Ukranian translation added
```
Comment (mazulo) [inline CHANGELOG.rst:1403]:
```suggestion
- Ukrainian translation added
```

File: docs/upgrade/3.10.1.rst [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+* Fixed #6413: migrations 0019 and 0020 on multi db setups (#6708) (826d57f0f) -- Petr Glotov
+* Added fix to migrations to handle multi database routing (#6721) (98658a909) -- Michael Anckaert
+* Fixed issue where default fallbacks is not used when it's an empty list (#6795) (5d21fa5eb) -- Arjan de Pooter
+* Fixed prefix_default_language = False redirect behavior (#6851) (34a26bd1b) -- Radek Stępień
+* Fix not checking slug uniqueness on page move (#6958) (5976d393a) -- Iacopo Spalletti
+* Fixed DontUsePageAttributeWarning message (#6734) (45383888e) -- carmenkow
+* Fixed Cache not invalidated when using a PlaceholderField outside the CMS #6912 (#6956) (3ce63d7d3) -- Benjamin PIERRE
+* Fixed unexpected behavior get_page_from_request (#6974) (#6073) (52f926e0d) -- Yuriy Mamaev
+* Fixed django treebeard 4.5.1 compatibility (#6988) (eeb86fd70) -- Aiky30
+* Fixed Bad Title.path in Multilanguage sites if parent slug is created or modified (#6968) (6e7b0ae48) -- fp4code
```
Comment (mazulo) [inline docs/upgrade/3.10.1.rst:41]:
```suggestion
* Fixed bad Title.path in Multilanguage sites if parent slug is created or modified (#6968) (6e7b0ae48) -- fp4code
```

File: docs/upgrade/3.10.1.rst [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+* Fixed redirect issues when i18n_patterns had prefix_default_language = False
+* Fixed not checking slug uniqueness when moving a page
+* Fixed builds on RTD
+* Fixed the cache not being invalidated when updating a PlaceholderField in a custom model
+* Fixed 66622 bad Title.path in multilingual sites when parent slug is created or modified
```
Comment (mazulo) [inline docs/upgrade/3.10.1.rst:46]: Question: would this refers to issue #6662?

File: docs/upgrade/3.10.1.rst [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+* Fixed 6973 bag with unexpected behavior ``get_page_from_request``
```
Comment (mazulo) [inline docs/upgrade/3.10.1.rst:47]: Question: would this refers to issue #6973?

**Review by mazulo — COMMENTED** [review]
I was just reviewed and everything looks good. Just left a few comments in some potential typos (non-blockers I guess)

File: docs/upgrade/3.10.1.rst (hunk 8 above)
Comment (marksweb) [reply docs/upgrade/3.10.1.rst:47]: Quite likely

**Comment by marksweb** [comment]
@mazulo thanks. There was some work done on a spell checker as part of the pre-commit hooks. Not sure if it merged yet but it should help prevent future spelling errors

**Review by mazulo — APPROVED** [review]

**Pull request merged**